ui-charts/src/manchette/styles/manchette.css

```diff
@@ -77,8 +93,6 @@
   }
 
   .space-time-chart-container {
-    height: 561px;
```

Comment:
Nice to be able to get rid of this hardcoded height :)

Reply:
There are still many hanging around though :sweat_smile: